Runtime/Async/Coroutine/CoroutineUtility.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections;
+using UnityEngine;
+
+namespace StansAssets.Foundation.Async
+{
+    /// <summary>
+    /// Coroutine helper methods and <see cref="MonoBehaviour"/> callbacks.
+    /// </summary>
+    public class CoroutineUtility
+    {
+        /// <summary>
+        /// Starts a Coroutine.
+        /// An equivalent for [MonoBehaviour.StartCoroutine](https://docs.unity3d.com/ScriptReference/MonoBehaviour.StartCoroutine.html)
+        ///
+        /// The execution of a coroutine can be paused at any point using the yield statement.
+        /// When a yield statement is used, the coroutine pauses execution and automatically resumes at the next frame.
+        /// See the Coroutines documentation for more details.
+        ///
+        /// Coroutines are excellent when modeling behavior over several frames.
+        /// The StartCoroutine method returns upon the first yield return, however you can yield the result,
+        /// which waits until the coroutine has finished execution.
+        /// There is no guarantee coroutines end in the same order they started, even if they finish in the same frame.
+        ///
+        /// Yielding of any type, including null, results in the execution coming back on a later frame, unless the coroutine is stopped or has completed.
+        ///
+        /// _Note:_ You can stop a coroutine using MonoBehaviour.StopCoroutine
```

Comment:
Looks like a typo... It doesn't seem true that these coroutines can be stopped with MonoBehaviour 🤔

Reply:
Yea that copy-paste error, will fix that.
But in general the different topic between `CoroutineUtility` and `MonoBehaviour` is covered above:

```
An equivalent for [MonoBehaviour.StartCoroutine](https://docs.unity3d.com/ScriptReference/MonoBehaviour.StartCoroutine.html)
```

Long story short, no difference :)